phpcs.xml.dist

```diff
@@ -45,6 +65,9 @@
         <properties>
             <property name="blank_line_check" value="true" />
         </properties>
+        <!-- Beans requires a new blank line BEFORE each control structure.
+             Excluding this rule will allow a blank line after all control structures. -->
+        <exclude name="WordPress.WhiteSpace.ControlStructureSpacing.BlankLineAfterStart"/>
```

Comment:
I don't believe these three lines are in sync with each other.

The `BlankLineAfterStart` check aims to remove the following blank line _inside_ the structure:

```php
if ( true ) {

    do something();
}
```

As such, it has no bearing on new lines _before_ a control structure (line before `if`), or after a control structure (line after closing `}`), which is what the two comment lines refer to.